### ISSUE TITLE
Chase.com login fields fail to detect as of version 1.3.2

### ISSUE THREAD

**Issue by tfvlrue** [keepassxreboot/keepassxc-browser#386]
## Expected Behavior

Login fields should be detected on all websites.

## Current Behavior

The login fields on https://www.chase.com/ are no longer detected as of version 1.3.2

## Possible Solution

Tested in Firefox 64.0.2 and Chrome 71. This looks like a regression in version 1.3.2. I reverted back to 1.3.1 in Firefox and the login fields were detected once I clicked the "Redetect login fields" button.

## Steps to Reproduce (for bugs)

1. Navigate to https://www.chase.com, making sure the browser is full screen
2. The login box will appear in the upper right after a second.
3. Notice the fields are not auto-detected.
4. Click the "Redetect login fields" button. Notice they are still not detected (password generation icon does not appear, and credentials do not auto-fill). In version 1.3.1, it worked.

## Debug info

KeePassXC - 2.41
KeePassXC-Browser - 1.3.2
Operating system: Windows 7
Browser: Chrome/Firefox

Thanks for looking into it. Let me know if there's any other info I can provide.

**Comment by droidmonkey** [comment]
I can confirm this as well

**Comment by varjolintu** [comment]
It's broken by https://github.com/keepassxreboot/keepassxc-browser/pull/371. I made a fix where I moved some of the behaviour to be executed in each frame. Some sites need this.

I also noticed that the site URL varies a lot. Even when you go to the frontpage, the actual URL for the login is always `secure05b.chase.com` or similar. This is why the entry URL should be `https://chase.com` so it will be handled as `https://*.chase.com`.
